pipenv/project.py

```diff
@@ -537,8 +487,11 @@ def parsed_pipfile(self):
 
         (call clear_pipfile_cache() afterwards if mutating)"""
         contents = self.read_pipfile()
-        # use full contents to get around str/bytes 2/3 issues
-        cache_key = (self.pipfile_location, contents)
+        # This should be pretty fast; we need this pipfile a lot
```

Comment:
This was the prior implementation of cache_key before someone realized it didn't work with py2 and made it be the entire contents.   Now that py2 is not supported, we can make this cache key short again.